sections/futures/Trade/ManagePosition.tsx

```diff
@@ -212,6 +213,23 @@ const PlaceOrderButton = styled(Button)`
 	height: 55px;
 	text-align: center;
 	white-space: normal;
+	color: ${(props) => props.theme.colors.selectedTheme.black};
+
+	&.long {
+		background: ${(props) => props.theme.colors.selectedTheme.green};
+	}
+	&.long:hover {
```

Comment:
Would be nice if we could add these styles to the base button so we don't have to repeat them in the various places.